example_profile.mobileconfig

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
+			<key>PayloadDisplayName</key>
+			<string>Nudge Configuration</string>
+			<key>PayloadIdentifier</key>
+			<string>com.erikng.nudge</string>
```

Comment:
Should this now reflect `com.macadmins.nudge` given that the repo has been moved into public domain?

Reply:
Hey @erikng 
What do you think about this change (and related changes below)?

Reply:
fixed in https://github.com/macadmins/nudge/pull/49/commits/ab7b5a8062b358c4bf8fa3d5d1d80d360e25e265